src/OrchardCore.Modules/OrchardCore.Workflows/Assets/Styles/workflow-canvas.scss

```diff
@@ -68,6 +68,14 @@
                 }
             }
         }
+        
+        span {
+            display: inline-block;
+            max-width: 300px;
```

Comment:
It doesn't prevent the box itself to not become too wide by doing this. I can find other type of activities/events that are not using span at all.

Reply:
Example : 

![image](https://user-images.githubusercontent.com/3228637/80280428-faaab600-86d1-11ea-9198-05e18af70e31.png)